### PR TITLE
LEAF-3118 Simplified interface for (workflow requirement) access management

### PR DESCRIPTION
**Pull request by mgaoVA**
This builds a list of active Workflow Requirements, tentatively labeled as "Task", and provides a simple interface to add/remove groups.

### CODE REVIEW

**Review by AmandaSumner — APPROVED** [review]

**Review by maxrichardjr — APPROVED** [review]

File: LEAF_Request_Portal/admin/templates/mod_access_matrix.tpl [hunk 1]
```diff
@@ -0,0 +1,243 @@
+<style>
```
Comment (raphaelfretter) [inline LEAF_Request_Portal/admin/templates/mod_access_matrix.tpl:1]: Apparent spacing issue with menu at the top on this page.

**Review by raphaelfretter — CHANGES_REQUESTED** [review]

File: LEAF_Request_Portal/admin/index.php [hunk 2]
```diff
@@ -435,6 +435,19 @@ function hasDevConsoleAccess($login, $db_phonebook)
...
+        $t_form = new Smarty;
+        $t_form->left_delimiter = '<!--{';
+        $t_form->right_delimiter = '}-->';
+
+        $main->assign('useUI', true);
+        $t_form->assign('CSRFToken', $_SESSION['CSRFToken']);
+
+        $main->assign('body', $t_form->fetch(customTemplate('mod_access_matrix.tpl')));
+
+        $tabText = 'Recover disabled fields';
```
Comment (aerinkayne) [inline LEAF_Request_Portal/admin/index.php:449]: tab/page title

File: LEAF_Request_Portal/admin/templates/mod_workflow.tpl [hunk 3]
```diff
@@ -1,4 +1,3 @@
-<script src="../../../libs/js/promise-pollyfill/polyfill.min.js"></script>
```
Comment (shaneodd) [inline LEAF_Request_Portal/admin/templates/mod_workflow.tpl:1]: This is in templates\orgChart_import.php and LEAF_import_data.tpl as well.

**Review by aerinkayne — CHANGES_REQUESTED** [review]
see specific comments regarding minor style adjustments and page title

File: LEAF_Request_Portal/admin/index.php (hunk 2 above)
Comment (mgaoVA) [reply LEAF_Request_Portal/admin/index.php:449]: Got a bit overzealous with copy/paste!

File: LEAF_Request_Portal/admin/templates/mod_access_matrix.tpl (hunk 1 above)
Comment (mgaoVA) [reply LEAF_Request_Portal/admin/templates/mod_access_matrix.tpl:1]: Nice catch

File: LEAF_Request_Portal/admin/templates/mod_access_matrix.tpl [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+#access_matrix {
+    margin: auto;
+    width: 80%;
+}
+
+.table td {
+    font-size: 1rem;
+}
+
+li {
```
[the remark this answers is not in the document]
Comment (mgaoVA) [reply LEAF_Request_Portal/admin/templates/mod_access_matrix.tpl:15]: Turns out that li selector was dead code -- started off as an <li> but turned into a <table>

File: LEAF_Request_Portal/admin/templates/mod_workflow.tpl (hunk 3 above)
Comment (mgaoVA) [reply LEAF_Request_Portal/admin/templates/mod_workflow.tpl:1]: Going to leave those for a future IE9 "decruft" task

**Review by aerinkayne — APPROVED** [review]

**Review by pete-nerantzinis — APPROVED** [review]

**Review by raphaelfretter — APPROVED** [review]

**Review by Pelentan — APPROVED** [review]

**Comment by AmandaSumner** [comment]
Site Distribution dropdown

Site Distribution Link in the Admin Dropdown does not work

**Pull request merged**